docs/README.md

```diff
@@ -0,0 +1,7 @@
+# Documentation
+
+* [statinamic.md](statinamic.md): Explains how Statinamic works and how you can edit the code and the content of `postcss.com`.
+
+* [react.md](react.md): A short introduction to React.js components with further links if you're new to React.
+
+* [getting-started.md](getting-started.md): How to set up a local copy of the website to make changes.
```

Comment:
I would reverse the order: getting started (so people have the "wow it's easy to start" effect), the "react" (because it's a trend), then "statinamic".